templates/admin/src/components/layout/Header.tsx

```diff
@@ -1,43 +1,56 @@
 import React from 'react';
 import { getGlobalInstance } from 'plume-ts-di';
 import { MenuItem } from '@mui/material';
+import LocaleService from '../../i18n/locale/LocaleService';
 import { Locale } from '../../lib/locale-resolver/LocaleResolver';
 import { User } from '../../services/session/User';
 import LocaleSelector from '../theme/LocaleSelector';
 import SessionService from '../../services/session/SessionService';
-import DropdownMenu from '../theme/DropdownMenu';
-import LocaleService from '../../i18n/locale/LocaleService';
 import MessageService from '../../i18n/messages/MessageService';
+import DropdownMenu from '../theme/DropdownMenu';
 
 type HeaderProps = {
   currentLocale: Locale;
   currentUser?: User;
 };
 
-export default function Header({ currentLocale, currentUser } : HeaderProps) {
+export default function Header({ currentLocale, currentUser }: HeaderProps) {
   const localeService = getGlobalInstance(LocaleService);
   const sessionService = getGlobalInstance(SessionService);
   const messages = getGlobalInstance(MessageService).t();
 
+  const getInitialLettersOfUser = () => {
+    const parts = currentUser?.fullName?.split(' ');
+    return parts?.map((part) => part.charAt(0)).join('');
+  };
+
   return (
     <header id="main-header">
-      <LocaleSelector
-        currentLocale={currentLocale}
-        availableLocales={localeService.getAvailableLocales()}
-        onLocaleSelected={(newLocale) => localeService.setCurrentLocale(newLocale)}
-      />
-      {
+      <h1 className="section_name">{messages['app.name']}</h1>
+      <div className="header_actions">
+        <div className="header_action">
+          <LocaleSelector
+            currentLocale={currentLocale}
+            availableLocales={localeService.getAvailableLocales()}
+            onLocaleSelected={(newLocale) => localeService.setCurrentLocale(newLocale)}
+          />
+        </div>
+        {
           currentUser
           && (
-            <DropdownMenu label={currentUser.fullName} id="user-menu">
-              <MenuItem
-                onClick={() => sessionService.disconnect()}
-              >
-                {messages['action.disconnect']}
-              </MenuItem>
-            </DropdownMenu>
+            <div className="header_action header_action--circle">
+              <DropdownMenu label={getInitialLettersOfUser() || ''} id="user-menu">
+                <div id="user-name">{currentUser.fullName}</div>
+                <MenuItem
+                  onClick={() => sessionService.disconnect()}
```

Comment:
onclick{sessionService.disconnect}

Reply:
Non pas ici. Derrière, le 'this' du service sera à undefined car il pointera au même 'this' que là ou la fonction a été appelée. C'est parce que la fonction 'disconnect' est une function et non une arrow function dans le service